internal/cmd/extensions/delete.go

```diff
@@ -0,0 +1,63 @@
+// Copyright Mia srl
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package extensions
+
+import (
+	"fmt"
+
+	"github.com/mia-platform/miactl/internal/client"
+	"github.com/mia-platform/miactl/internal/clioptions"
+
+	"github.com/spf13/cobra"
+)
+
+func addExtensionIDRequiredFlag(options *clioptions.CLIOptions, cmd *cobra.Command) {
+	flags := cmd.Flags()
+	flags.StringVarP(&options.EntityID, "extension-id", "i", "", "the extension id that should be deleted")
```

Comment:
I don't think that `i` is a meaningful abbreviation for `extension-id` can we avoid to set it?

Reply:
I've followed the abbreviation use for the marketplace item id but ok, i'm removing it